--- a/knot.spec
+++ b/knot.spec
@@ -49,2 +49,2 @@
-BuildRequires:pkgconfig(hiredis)
+# BuildRequires:pkgconfig(hiredis)
 %if 0%{?fedora} || 0%{?rhel}
@@ -51,1 +51,1 @@
-BuildRequires:softhsm
+# BuildRequires:softhsm
@@ -57,1 +57,1 @@
-BuildRequires:pkgconfig(libmaxminddb)
+# BuildRequires:pkgconfig(libmaxminddb)
@@ -61,1 +61,1 @@
-BuildRequires:  python3-devel
+# BuildRequires:  python3-devel
@@ -67,1 +67,1 @@
-BuildRequires:  python3-setuptools
+# BuildRequires:  python3-setuptools
@@ -81,1 +81,1 @@
-BuildRequires:python3-sphinx
+# BuildRequires:python3-sphinx
